### DISCUSSION TITLE
Default to Dashboard if Signed In

### DISCUSSION THREAD

**Discussion by fadingmorsecode** [TheOdinProject/theodinproject#4536] [Ideas]
One thing I have found annoying is being directed to the main introductory page for TOP despite me being signed in and wanting to resume a lesson quickly. 

Please redirect to dashboard if a user is signed into their account.

**Comment by CouchofTomato** [comment]
Hey @fadingmorsecode 

Directed at what stage? Are you referring to when you visit the homepage, you want to be auto directed to the dashboard if you're signed in?

**Reply by fadingmorsecode** [reply]
That's exactly what I would like to happen instead of the homepage that's there currently
